src/lib/components/schedule/schedules-table-row.svelte

```diff
@@ -52,12 +51,7 @@
     <WorkflowStatus status={schedule?.info?.paused ? 'Paused' : 'Running'} />
   </td>
   <td class="cell whitespace-pre-line break-words">
-    <Link
-      class="text-base"
-      on:click={() => {
-        goto(route);
-      }}
-    >
+    <Link class="text-base" href={route}>
```

Comment:
So why I did this...

With Link href={}, it does a full reload and refetches settings/cluster....

With on:click and goto, it does not

Reply:
So the question begs.  How do keep a tags accessible (hrefs) but use client-side routing with goto

Reply:
Whelp, I'm gonna add that back then 😅 Changed this before I updated the `Link` component, but now it should still be tabbable.